controllers/fenceagentsremediation_controller_test.go

```diff
@@ -285,7 +286,7 @@ func cleanupTestedResources(va1, va2 *storagev1.VolumeAttachment, pod *corev1.Po
 	podTest := &corev1.Pod{}
 	if err := k8sClient.Get(context.Background(), client.ObjectKeyFromObject(pod), podTest); err == nil {
 		log.Info("Cleanup: clean pod", "pod name", podTest.Name)
-		Expect(k8sClient.Delete(context.Background(), podTest)).To(Succeed())
+		Expect(k8sClient.Delete(context.Background(), podTest, forced)).To(Succeed())
```

Comment:
Would you do the same for "test-pod" in e2e tests?

Reply:
So, maybe doing both in the next PR? Using this one only for randomization?

Reply:
SGTM :D https://github.com/medik8s/fence-agents-remediation/pull/110